fhir-audit/src/test/java/testng.xml

```diff
@@ -1,5 +1,3 @@
-<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
```

Comment:
please undo this. 
it puts a warning in the build.

Reply:
ha, OK. I removed it because I found there is no such header in the testng.xml of the other 2 testng.xml files

Reply:
i have added it to one more from my PR thats been opened for a while.  but if you know where those are, you can just add it in on all of them

Reply:
Yeah, it's in a couple of spots 

```
./fhir-server-test/src/test/java/testng.xml
./fhir-persistence-jdbc/src/test/java/testng.xml
```

You added fhir-server-test. 

The only one that remains is fhir-persistence-jdbc